tests/transforms/test_temporal.py

```diff
@@ -41,3 +41,22 @@ def test_rearrange_integration_in_augmentation_sequential() -> None:
     out_batch = train_transforms(batch)
     assert 'image' in out_batch
     assert out_batch['image'].shape == (b, t * c, h, w)
+
+
+def test_temporal_embedding_shape_and_values() -> None:
```

Comment:
Let's also test the values themselves. They should be bounded between [-1, 1]. It may be useful to check `math.isclose(output, expected)` for a real number to make sure we do the `pd.Timestamp <-> float` conversion correctly, as I noticed that it isn't exactly intuitive and can be bug prone.